GliaWidgets/Resources/en.lproj/Localizable.strings

```diff
@@ -202,3 +203,5 @@
 "upgrade.video.one_way.title" = "{operatorName} has offered you to see their video";
 "upgrade.video.two_way.title" = "{operatorName} has offered you to upgrade to video";
 "visitor_code.failed" = "Could not load the visitor code. Please try refreshing.";
+"live_observation.confirm.title" = "Before We Continue";
+"live_observation.confirm.message" = "Please allow the {company name} representative to view this application screen for improved support.";
```

Comment:
Please update company name marker. Karl already updated it in custom locales PR
```suggestion
"live_observation.confirm.message" = "Please allow the {companyName} representative to view this application screen for improved support.";
```

Reply:
fixed [912c983](https://github.com/salemove/ios-sdk-widgets/pull/785/commits/912c983b6d9a6585b628dc8820cb19c7f77c25af)